python/lancedb/remote/table.py

```diff
@@ -72,7 +72,17 @@ def create_index(
         replace: bool = True,
         accelerator: Optional[str] = None,
     ):
-        raise NotImplementedError
+        index_type = "vector"
+
+        data = {
+            "column": vector_column_name,
+            "index_type": index_type,
+            "metric_type": metric,
+        }
+        resp = self._conn._loop.run_until_complete(
+            self._conn._client.post(f"/v1/table/{self._name}/create_index/", data=data)
+        )
+        return resp
 
```

Comment:
can we add a test for this API ?

Reply:
Added the test, but for it to run properly on gha we might need to add the env variables to github secrets, which I don't think I have perms to do
Also attempted adding the test to the integration tests instead, but that didn't seem to want to use the non-published version of lancedb

Reply:
I don't think we've generally tested the remote implementations in this test suite. This is a limitation of the node SDK as well.